tests/forms_tests/widget_tests/test_checkboxselectmultiple.py

```diff
@@ -173,3 +176,29 @@ class TestForm(forms.Form):
         bound_field = TestForm()['f']
         self.assertEqual(bound_field.field.widget.id_for_label('id'), '')
         self.assertEqual(bound_field.label_tag(), '<label>F:</label>')
+
+    @override_settings(USE_L10N=True, USE_THOUSAND_SEPARATOR=True)
```

Comment:
I'd order the test with the other `check_html()` tests (above `test_use_required_attribute`).